assets/src/application/ui/layout/EspressoDropdownMenu/types.ts

```diff
@@ -0,0 +1,13 @@
+import { AnyObject } from '@appServices/utilities/types';
+
+export interface DropdownMenuProps {
+	children(props: AnyObject): JSX.Element;
+	className?: string;
+	controls?: [];
+	icon?: string | null;
+	menuProps?: AnyObject;
+	label: string;
+	popoverProps?: AnyObject;
+	toggleProps?: AnyObject;
+	[key: string]: any;
```

Comment:
You could use `DropdownMenuProps  extends AnyObject` and remove this line here

Reply:
Done, thx TS-master :bowing_man: